### DISCUSSION TITLE
Issue with certificates from https://esmatlas.com

### DISCUSSION THREAD

**Discussion by valasatava** [facebookresearch/esm#665] [ESM Metagenomic Atlas]
We have encountered an issue while attempting to establish a connection between our Java client and the `https://esmatlas.com` API services. Specifically, we are consistently encountering the following error message:

`Fatal (CERTIFICATE_UNKNOWN): PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target`

This issue appears to persist across environments utilizing OpenJDK 17 and OpenJDK 21.

Could spmeone kindly provide insights into the potential causes of this error and any recommended steps or best practices for mitigating it? Additionally, if there are any specific configurations or certificates required to establish a secure connection with `esmatlas.com`, we would greatly appreciate guidance on how to proceed.

**Comment by josemduarte** [comment]
This issue happens with curl too, this is the error message:
```
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.

```

This cert checker website does bring up a warning: https://www.sslshopper.com/ssl-checker.html#hostname=https://esmatlas.com
